class-devices.php

```diff
@@ -311,21 +311,10 @@ public function remove_device() {
 	 */
 	public function custom_user_profile_fields( $profileuser ) {
```

Comment:
https://developer.wordpress.org/reference/hooks/edit_user_profile/
```suggestion
	public function custom_user_profile_fields( $profile_user ) {
```